pagopa/io-app
# chore(Cross): [IOAPPX-434] Update the main icon app for iOS + Add support for dark and tinted appearances

## Short description
This PR updates the main icon app for iOS and adds support for dark and tinted appearances. Check out the video below for a preview.

## List of changes proposed in this pull request
- Add three different icon assets at `1024×1024` size for default, dark and tinted appearances
- Update icon in the main README

### Preview

https://github.com/user-attachments/assets/e02b6533-9cc1-4cb8-9319-9537e1851145



## How to test
1. Run the app in the local environment
2. Go to **Customize…** option through long press on the home screen
3. Change visual appearance
4. Check if the optical alignment is correct (for visual designers only 👀)
